src/main/java/com/touchtunes/abtestingpoc/service/abtest/ABTastyImpl.java

```
@@ -27,7 +27,7 @@ public boolean isMammalFilter() {
 			log.info("ExecutionException: {}", e);
 			throw new RuntimeException(e);
```

it's the question here. If Flagship as a pletform will have some technical problem that means we will have errors and as result exception. Do we want to rethrow this exception which will effect all users or better just to log this exception (and monitor it via sumologic) and let endpoint behave normally?